lib/components/private/Modal/ModalContent.tsx

```diff
@@ -0,0 +1,248 @@
+            ]}
+          >
+            <Stack space="large">
+              {illustration ? (
+                <Stack space="medium" align="center">
+                  <Box paddingX="gutter">{illustration}</Box>
+                  <ModalContentHeader
+                    title={title}
+                    headingLevel={headingLevel}
+                    description={description}
+                    descriptionId={descriptionId}
+                    center={Boolean(illustration)}
+                    ref={headingRef}
+                  />
+                </Stack>
+              ) : (
+                <Columns space="none">
+                  <Column>
+                    <ModalContentHeader
+                      title={title}
+                      headingLevel={headingLevel}
+                      description={description}
+                      descriptionId={descriptionId}
+                      center={Boolean(illustration)}
+                      ref={headingRef}
+                    />
+                  </Column>
+                  <Column width="content">
+                    <Box width="touchable" />
+                  </Column>
+                </Columns>
+              )}
+              <Fragment>{children}</Fragment>
+            </Stack>
+          </Box>
+        </RemoveScroll>
+        <Box
+          position="absolute"
+          zIndex="sticky"
+          top={0}
+          right={0}
+          paddingTop={modalPadding}
+          paddingRight={modalPadding}
+        >
+          <Box
+            className={[
+              useNegativeMarginTop('xsmall'),
+              styles.negativeMarginRightXSmall,
+            ]}
+          >
+            <Box
+              position="relative"
+              className={styles.cropIconSpace[headingLevel]}
+            >
+              <Box
+                component="button"
+                aria-label={closeLabel}
+                borderRadius="full"
+                background="card"
+                padding="xsmall"
+                cursor="pointer"
+                position="relative"
+                onClick={onClose}
+                outline="none"
+                transition="touchable"
+                className={[
+                  styles.closeButtonRoot,
+                  styles.pointerEventsAll,
+                  useVirtualTouchable(),
+                ]}
+              >
+                <Overlay
+                  boxShadow="outlineFocus"
+                  transition="fast"
+                  onlyVisibleForKeyboardNavigation
+                  borderRadius="full"
+                  className={styles.closeButtonFocus}
+                />
+                <Box
+                  position="relative"
+                  transition="fast"
+                  className={[
+                    styles.closeButtonOpacity,
+                    styles.closeIcon[headingLevel],
+                  ]}
+                >
+                  <IconClear size="fill" aria-hidden />
```

Comment:
I dont think we need to mark this as hidden. We dont seem to do it for other icons elsewhere.